com.unity.formats.alembic/Runtime/Scripts/Importer/Behaviours/AlembicCustomData.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+
+namespace UnityEngine.Formats.Alembic.Importer
+{
+    /// <summary>
+    /// Class that stores ancillary data for the current Alembic Node.
```

Comment:
I didn't even know the word "ancillary" existed (learning something every day 😸).
Although I think it's correct, I would probably use more generic terminology such as "additional", for example.